lib/cucumber/glue/dsl.rb

```diff
@@ -136,5 +136,7 @@ def register_rb_step_definition(regexp, symbol = nil, options = {}, &proc)
   end
 end
 
-# TODO: can we avoid adding methods to the global namespace (Kernel)
-extend(Cucumber::Glue::Dsl)
+class Object
```

Comment:
You can do this, but part of the config file is where you specify overrides,

So you could ignore this rule on this file, and stipulate why

Reply:
It seems to work like that. Test are passing well. Is there some cases where it could fail?

Reply:
The only difference for this would come in a situation whereby someone is using BOTH a user-defined version of these method / lVar names and the cucumber ones. We shouldn't really look to cover that situation.

But what I'm saying here, is instead of us just tricking rubocop, lets leave it as is, but document it in rubocop.yml by saying "This allows users to have nice DSL defined methods like ParameterType and Before in their support directory, instead of needing to instantiate new objects."